doc/services/pm/device.rst

```diff
@@ -231,6 +231,22 @@ support in a device driver.
         PM_DEVICE_DT_INST_GET(0), NULL, NULL, POST_KERNEL,
         CONFIG_KERNEL_INIT_PRIORITY_DEFAULT, NULL);
 
+.. _pm-device-shell:
+
+Device PM Testing
+*****************
```

Comment:
Nit: (I wouldn't repeat "Device PM" as we're already in the Device PM section

```suggestion
Shell commands
**************
```

Reply:
done